plugin.yml

```diff
@@ -11,4 +11,4 @@ commands:
 permissions:
   f.command:
     default: true
-    description: "Allow use of all faction commands"
\ No newline at end of file
+    description: "Allow Players Of use Faction Commands/SubCommands"
```

Comment:
"Players of use factions commands/subcommands" is not proper english at all...
Lowercase P as its the second word in the sentence. "of use" is not english..